UnityProject/Assets/Scripts/Managers/AdminCommandsManager.cs

```diff
@@ -1187,5 +1185,94 @@ private void ColourTile(MatrixInfo matrixInfo, Vector3Int localPos, LayerType la
 		}
 
 		#endregion
+
+		#region DebugCommands
+
+		private static IEnumerator KillLights()
+		{
+			if (MatrixManager.MainStationMatrix?.Objects == null) yield break;
+
+			var currentIndex = 0;
+			var maximumIndexes = 20;
+			foreach (var stationObject in MatrixManager.MainStationMatrix.Objects.GetComponentsInChildren<LightSource>())
+			{
+				if (currentIndex >= maximumIndexes)
+				{
+					currentIndex = 0;
+					yield return WaitFor.EndOfFrame;
+				}
+				stationObject.Integrity.ForceDestroy();
+				currentIndex++;
+			}
+		}
+
+		private static IEnumerator SelfPowerEverything()
+		{
+			if (MatrixManager.MainStationMatrix?.Objects == null) yield break;
+
+			var currentIndex = 0;
+			var maximumIndexes = 20;
+			foreach (var stationObject in MatrixManager.MainStationMatrix.Objects.GetComponentsInChildren<APCPoweredDevice>())
+			{
+				if (currentIndex >= maximumIndexes)
+				{
+					currentIndex = 0;
+					yield return WaitFor.EndOfFrame;
+				}
+				stationObject.ChangeToSelfPowered();
+				currentIndex++;
+			}
+		}
+
+		private static IEnumerator TurnOnAllEmergancyLights()
+		{
+			if (MatrixManager.MainStationMatrix?.Objects == null) yield break;
+
+			var currentIndex = 0;
+			var maximumIndexes = 20;
+			foreach (var stationObject in MatrixManager.MainStationMatrix.Objects.GetComponentsInChildren<LightSource>())
+			{
+				if (currentIndex >= maximumIndexes)
+				{
+					currentIndex = 0;
+					yield return WaitFor.EndOfFrame;
+				}
+				stationObject.ServerChangeLightState(LightMountState.Emergency);
+				currentIndex++;
+			}
+		}
+
+
+		[Command(requiresAuthority = false)]
+		public void DestroyAllLights(NetworkConnection player)
```

Comment:
```suggestion
		public void DestroyAllLights(NetworkConnectionToClient sender = null)
```

You do not need to send the parameter, Mirror auto fills it